paddle/CMakeLists.txt

```diff
@@ -17,10 +17,10 @@ add_subdirectory(strings)
 
 find_package(Boost QUIET)
 
+
 if(Boost_FOUND)
```

Comment:
if(Boost_FOUND)
**include_directories(${Boost_INCLUDE_DIRS})**
add_subdirectory(platform)
add_subdirectory(framework)
endif()

We need this line to guarantee the compiler can find the header file.